lib/reports/ppl-details/index.js

```diff
@@ -15,7 +15,6 @@ module.exports = ({ db }) => {
         db.asl('project_versions')
           .where('project_versions.status', 'granted')
           .whereNull('deleted')
-          .groupBy('project_versions.project_id', 'project_versions.id') // group by limits results to 1 per project
```

Comment:
This line was not having the effect the comment suggested, and removing it makes no obvious difference to the functionality.

Reply:
It won't make any diff while there's a `first()` in the query as well - I think the issue was the `, 'project_versions.id'` bit, but sounds like it's working ok with the `first()`.